fixtures/mega-lite/mw1.json

```diff
@@ -0,0 +1,521 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "MW1",
+  "categories": [
+    "Moving Head",
+    "Color Changer"
+  ],
+  "meta": {
+    "authors": ["Ryan Goodwin"],
+    "createDate": "2024-10-06",
+    "lastModifyDate": "2024-10-06"
+  },
+  "links": {
+    "manual": [
+      "https://www.mega.lighting/_files/ugd/21ff82_e921a48d91974862860f4e5f05ed7f30.pdf"
+    ],
+    "productPage": [
+      "https://www.mega.lighting/mw1"
+    ],
+    "video": [
+      "https://www.youtube.com/watch?v=StuUWiJoji8",
+      "https://www.youtube.com/watch?v=pS-_VBK4j8k"
+    ]
+  },
+  "physical": {
+    "dimensions": [323.85, 228.6, 393.7],
+    "weight": 8.6,
+    "power": 376,
+    "powerConnectors": {
+      "powerCON": "input and output"
+    },
+    "DMXconnector": "3-pin and 5-pin",
+    "bulb": {
+      "type": "7x Osram RGBW 40W LED"
+    },
+    "lens": {
+      "degreesMinMax": [9, 44]
+    }
+  },
+  "matrix": {
+    "pixelKeys": [
+      [
+        [null, "7", null, "2", null],
+        ["6", null, "1", null, "3"],
+        [null, "5", null, "4", null]
+      ]
+    ],
+    "pixelGroups": {
+      "All": "all"
+    }
+  },
+  "templateChannels": {
+    "Red $pixelKey": {
+      "highlightValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Red"
+      }
+    },
+    "Green $pixelKey": {
+      "highlightValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Green"
+      }
+    },
+    "Blue $pixelKey": {
+      "highlightValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "Blue"
+      }
+    },
+    "White $pixelKey": {
+      "highlightValue": 0,
+      "capability": {
+        "type": "ColorIntensity",
+        "color": "White"
+      }
+    }
+  },
+  "availableChannels": {
+    "Pan": {
+      "fineChannelAliases": [
+        "Pan fine"
+      ],
+      "defaultValue": 32896,
```

Comment:
```suggestion
```

This value isn't quite half (it's about 50.2%), but generally we just leave defaults blank for pan/tilt.  That's a common enough parameter that control software should know to pick the halfway point.

Reply:
I am pretty sure I had found this default on another fixture as the Pan, Tilt, P/T Speed, and Dimmer were copy and pasted from another fixture in the repo (can't remember which one off the top of my head).

Reply:
Actually you could use the string "50%" in this case to make the intent more clear.